include/event2/http.h

```diff
@@ -487,6 +498,18 @@ enum evhttp_cmd_type {
 	EVHTTP_REQ_PATCH   = 1 << 8
 };
 
+#define EVHTTP_REQ_MAX EVHTTP_REQ_PATCH
+
+/** structure to allow users to define their own HTTP methods
+ * @see evhttp_set_extended_methods */
+struct evhttp_extended_method {
+	const char * method;
+	ev_uint16_t type;
+	ev_uint16_t flags;	/* Available flag : EVHTTP_METHOD_HAS_BODY */
```

Comment:
if type is limited to 9 bits, it's not possible to add any extended methods are there are already 9 standard methods !